FloatLabelFields/FloatLabelTextField.swift

```diff
@@ -144,7 +144,7 @@ import UIKit
 	
 	// MARK:- Private Methods
 	fileprivate func setup() {
-		borderStyle = UITextBorderStyle.none
+        borderStyle = UITextField.BorderStyle.none
```

Comment:
```suggestion
        borderStyle = .none
```
this use is correct :)